keploy.sh

```diff
@@ -71,20 +71,25 @@ installKeploy (){
     }
 
     install_docker() {
-        if ! docker network ls | grep -q 'keploy-network'; then
-            docker network create keploy-network
-        fi
+    if ! docker network ls | grep -q 'keploy-network'; then
+        docker network create keploy-network
+    fi
 
-        if [ "$OS_NAME" = "Darwin" ]; then
-            if ! docker volume inspect debugfs &>/dev/null; then
-                docker volume create --driver local --opt type=debugfs --opt device=debugfs debugfs
-            fi
-            alias keploy='sudo docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v $(pwd):$(pwd) -w $(pwd) -v /sys/fs/cgroup:/sys/fs/cgroup -v debugfs:/sys/kernel/debug:rw -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v '"$HOME"'/.keploy-config:/root/.keploy-config -v '"$HOME"'/.keploy:/root/.keploy --rm ghcr.io/keploy/keploy'
+    if [ "$OS_NAME" = "Darwin" ]; then
+        if ! docker volume inspect debugfs &>/dev/null; then
+            docker volume create --driver local --opt type=debugfs --opt device=debugfs debugfs
+        fi
+        alias keploy='docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v $(pwd):$(pwd) -w $(pwd) -v /sys/fs/cgroup:/sys/fs/cgroup -v debugfs:/sys/kernel/debug:rw -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v '"$HOME"'/.keploy-config:/root/.keploy-config -v '"$HOME"'/.keploy:/root/.keploy --rm ghcr.io/keploy/keploy'
+    else
```

Comment:
use else if here, we are doing this specifically on linux

Reply:
Kindly check if this was the intended change